pkg/controller/volume/attachdetach/attach_detach_controller_test.go

```diff
@@ -46,6 +46,7 @@ func Test_NewAttachDetachController_Positive(t *testing.T) {
 		informerFactory.Core().V1().PersistentVolumes(),
 		informerFactory.Storage().V1beta1().CSINodes(),
 		informerFactory.Storage().V1beta1().CSIDrivers(),
```

Comment:
Can you add a unit test for the new behavior?

Reply:
Sure, I would add some unit tests.

But before that, I should solve the conflicts first